client/src/components/Application/Application.js

```diff
@@ -1,13 +1,13 @@
-import React, {Component} from 'react';
```

Comment:
Remove all of these changes, or you will have a merge conflict. Just revert the changes if you can.